### DISCUSSION TITLE
LDAP login -> Wrong 2FA/TOPO code - but it is not acitivated

### DISCUSSION THREAD

**Discussion by blu-IT** [YetiForceCompany/YetiForceCRM#15180] [General]
Hello,

I have problems with the LDAP login. YF is configured to fetch login from a LDAP server, but the login always fails.
/cache/logs/*.log does not show any ldap error (debug is activated) only in YF admin backend I can see "failed login" for the user because of **Wrong 2FA/TOPO code** -  but I **have not** configured users to use 2FA.

Any ideas?

Thanks!

**Comment by mariuszkrzaczkowski** [comment]
version ?

**Comment by blu-IT** [comment]
Sorry, it is YF ver. 6.2.0 with latest SP 3.

**Reply by drakub** [reply]
Was this issue resolved? I also face a similar challenge?

**Reply by blu-IT** [reply]
Unfortunately not :-/
